internal/utils/testutil/utils.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2021 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
```

Comment:
i think the filename `utils.go` is a bit generic but couldn't think about a assertive name

Reply:
I have the same felling...

Maybe generic functions that can't have a specific file name can leave on this generic file.  What do you think @nathanmartinszup ?

Reply:
Maybe we can move the function to the path.go file or create a file for directory functions.

Reply:
I think that path would be good. I'll move.